Windows build errors in filament_framegraph_test.

**Describe the bug**
Attempting to build filament on Windows with tests enabled, gets build errors:

`..\filament\test\filament_framegraph_test.cpp(237): error C2059: syntax error: 'template'
D:\dev\filament\filament\src\fg2/FrameGraphPass.h(69): note: see reference to function template instantiation 'auto FrameGraphTest_WriteWrite_Test::TestBody::<lambda_2>::operator ()<Data>(const filament::FrameGraphResources &,const _T1 &,filament::backend::DriverApi &) const' being compiled
        with
        [
            Data=FrameGraphTest_WriteWrite_Test::TestBody::PassData,
            _T1=FrameGraphTest_WriteWrite_Test::TestBody::PassData
        ]`

and

`..\filament\test\filament_framegraph_test.cpp(238): error C2181: illegal else without matching if
..\filament\test\filament_framegraph_test.cpp(237): error C2065: 'gtest_ar': undeclared identifier
..\filament\test\filament_framegraph_test.cpp(237): error C2440: '<function-style-cast>': cannot convert from 'initializer list' to 'testing::internal::AssertHelper'`

**To Reproduce**
Steps to reproduce the behavior:
1.In a Windows VS2019 powershell terminal, with git, python, cmake, ninja in the PATH
2. run commands: 
`git clone https://github.com/google/filament.git
cd .\filament\
mkdir build-dest
cd .\build-dest\
cmake -GNinja ..
ninja`

Observer build failure

**Expected behavior**

Build should succeed.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - GPU: AMD RX5800

**Additional context**

Python 3.7.0
Microsoft (R) C/C++ Optimizing Compiler Version 19.28.29913 for x64
Microsoft (R) Incremental Linker Version 14.28.29913.0
ninja 1.10.1
cmake version 3.19.6

--- a/filament/test/filament_framegraph_test.cpp
+++ b/filament/test/filament_framegraph_test.cpp
@@ -210,7 +210,7 @@ TEST_F(FrameGraphTest, ReadRead) {
                 builder.sideEffect();
             },
             [=](FrameGraphResources const& resources, auto const& data, backend::DriverApi& driver) {
-                EXPECT_EQ(resources.template getUsage(data.input), FrameGraphTexture::Usage::SAMPLEABLE | FrameGraphTexture::Usage::COLOR_ATTACHMENT);
+                EXPECT_EQ(resources.getUsage(data.input), FrameGraphTexture::Usage::SAMPLEABLE | FrameGraphTexture::Usage::COLOR_ATTACHMENT);
             });
 
     EXPECT_TRUE(fg.isAcyclic());
@@ -234,7 +234,7 @@ TEST_F(FrameGraphTest, WriteWrite) {
                 EXPECT_EQ(data.output1, data.output2);
             },
             [=](FrameGraphResources const& resources, auto const& data, backend::DriverApi& driver) {
-                EXPECT_EQ(resources.template getUsage(data.output2), FrameGraphTexture::Usage::UPLOADABLE | FrameGraphTexture::Usage::COLOR_ATTACHMENT);
+                EXPECT_EQ(resources.getUsage(data.output2), FrameGraphTexture::Usage::UPLOADABLE | FrameGraphTexture::Usage::COLOR_ATTACHMENT);
             });
 
     fg.present(pass->output2);

